libsel4/include/interfaces/sel4.xml

```diff
@@ -810,9 +810,8 @@
                 used for TLS can only be written to in privilidged mode
              </brief>
              <description>
+                An invocation for setting TLSBase on all platforms. Threads modify their own Thread Local Storage (TLS) address using seL4_SetTLSBase across all platforms where they would not otherwise be able to modify the designated TLS register directly. This ensures that across all platforms, the TLS Base register is viewed as being completely mutable by the executing thread, just like all of the general purpose registers.
```

Comment:
I would limit the brief description to just "Set the TLS base of the target TCB" and don't assume the caller is the target thread. In general it's the thread launching another thread that sets the TLS, not the thread itself. Something like:

```suggestion
                An invocation for setting the Thread Local Storage (TLS) base address. This ensures that across all platforms, the TLSBase register is viewed as being completely mutable, just like all of the general purpose registers, even on platforms where modification is a privileged operation.
```